nuclio_sdk/test/test_logger.py

```diff
@@ -65,6 +65,14 @@ def test_log_with_date(self):
             self._io.getvalue(),
         )
 
+    def test_log_nan(self):
+        self._logger.info_with("logNaN", nan=float("NaN"))
+        self.assertIn("logNaN", self._io.getvalue())
+        self.assertIn(
+            '"with": {"nan": "NaN"}',
+            self._io.getvalue(),
+        )
+
```

Comment:
test inf/-inf as well while you're here?
Any other weird type like that?

Reply:
will do and not that i can think of